src/components/CategoryFilter.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
**Required**:`CategoryFilter` should be a presentational component, it should not connect to the Redux store.

https://redux.js.org/basics/usage-with-react#presentational-and-container-components

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/presentationalCategoryFilter.png)